test/micro/org/openjdk/bench/java/lang/StringBuilders.java

```diff
@@ -224,6 +233,47 @@ public String toStringCharWithInt8() {
         return result.toString();
     }
 
+    @Benchmark
+    public int appendWithIntLatin1() {
+        StringBuilder buf = sbLatin1;
+        buf.setLength(0);
+        for (long l : longArray) {
```

Comment:
I believe this should be
```suggestion
        for (int l : intsArray) {
```